FrontDesk/src/BlazorShared/Models/Room/RoomDto.cs

```diff
@@ -6,5 +6,6 @@ public class RoomDto
     {
         public int RoomId { get; set; }
         public string Name { get; set; }
+        public string Color { get; set; }
```

Comment:
Not sure this needs a color property. If you have a class in CSS like room1, room2, room3 then you could style each room just in the CSS and not need it in your shared model.